lib/puppet/provider/pulp_rpmbind/consumer.rb

```diff
@@ -4,7 +4,7 @@
   desc 'Bind/unbind to an RPM repo'
 
   confine osfamily: :redhat
-  commands consumer: '/bin/pulp-consumer'
+  commands consumer: 'pulp-consumer'
   commands grep:     '/bin/grep'
```

Comment:
Should we also change this to just grep?

Reply:
You *could* and it wouldn't hurt, but I didn't as it's `/bin/grep` everywhere I think.  Also, I didn't touch the other providers because running a pulp *server* certainly isn't support on EL6.

For anyone interested/googling, with pulp 2.14 repo on CentOS 6.9, I also needed the packages from
* https://copr.fedorainfracloud.org/coprs/g/qpid/qpid/
* https://copr.fedorainfracloud.org/coprs/dgoodwin/subscription-manager/

to meet all the dependencies.